blog/news-and-updates/team-spotlight-ii.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Team spotlight - Internet Identity
+description: Welcome to this installment of team spotlight. This week, the have a very exciting team to dive into - the Internet Identity team!
+tags: [Team spotlight]
+image: /img/blog/dev-update-blog-internet-id.jpg
+---
+
+![Internet Identity Team spotlight](../../static/img/blog/dev-update-blog-internet-id.jpg)
+
+Greetings developers! Welcome to this installment of team spotlight. This week, the have a very exciting team to dive into: the Internet Identity team! As the team behind developing one of the most cutting-edge tools for digital identities, their mission and focus is a personal favorite. Without further adieu, let's get started! 
+
+
+**Hello and welcome to this week's issue of DFINITY's team spotlight! This week, we're going to learn more about the team behind Internet Identity and what's in the works for digital identity on the Internet Computer.**
```

Comment:
Is it intentional to reiterate the paragraph above here?

Reply:
It was, as I tried to provide both a 'written' into and a 'speaking' intro to signify the beginning of the 'interview', but it does read a bit redundant. I've removed the second paragraph to fix this.